src/index.js

```diff
@@ -141,7 +121,10 @@ export const getTargets = (targets = {}) => {
 
   // Rewrite Electron versions to their Chrome equivalents
   if (targetOps.electron) {
-    targetOps.chrome = electronVersionToChromeVersion(targetOps.electron);
+    targetOps.chrome = parseInt(electronToChromium(targetOps.electron), 10);
```

Comment:
If we have for ex:
```js
"targets": {
  "chrome": 50,
  "electron": 1.3 // 52
}
```
looks like it will just rewrite chrome version with higher. So the result will be chrome: **52**, but not **50**.